core/domain/stats_domain_test.py

```diff
@@ -119,49 +127,63 @@ def test_to_dict(self):
         }
         observed_exploration_stats = self._get_exploration_stats_from_dict(
             expected_exploration_stats_dict)
+        # Using ignore[arg-type] because assertDictEqual method
+        # expects both arguments to be of type Dict[Any, Any]
```

Comment:
@vojtechjelinek @hardikkat24  is this correct? I would have thought a subset of Any could be passed into assertDictEqual.

Reply:
`TypedDict` is not a subset of `Dict[Any, Any]` (https://github.com/python/mypy/issues/4976), one other possibility here would be to use `cast`.

Reply:
Would it be preferred to use `cast` instead of `ignore[arg-type]`?